### ISSUE TITLE
Contribution guidelines

### ISSUE THREAD

**Issue by codewithsk** [jermainewang/dgl#281]
## ❓ Questions and Help

Is there a dev list/roadmap or feature tracker that contributors can use when deciding what to contribute to the library?

**Comment by jermainewang** [comment]
Yes, a more detailed breakdown on what's on our roadmap and how to contribute is on the way. Broadly speaking, there are following directions:
* More GNN models.
* More backend support. (currently we have Pytorch and MXNet)
* More platform support. (e.g. py2 support for many researchers who relies on legacy codes)
* More features. (such as more builtin functions)
* Document improvement.

You could raise an issue using "Feature request" so we can have more discussion about the specific plan.

P.S. we suggest move questions to our discuss forum here: [discuss.dgl.ai](https://discuss.dgl.ai).
